### ISSUE TITLE
Please add Orchard among blog providers

### ISSUE THREAD

**Issue by Piedone** [OpenLiveWriter/OpenLiveWriter#460]
Orchard (http://orchardproject.net/) also supports LiveWriter, so please add it as a provider. Tell if you need assistance.

**Comment by rfulwell** [comment]
I am new to this project and I just picked this issue as an easy one to get my feet wet. Orchard is probably the best known CMS in .NET/C# as far as I know. Is this a reasonable item to call out at the top level of the UI?

My changes are ready for a pull request having been tested slightly on my machine when adding a new blog. If you have tips on making it easier to go through the initial user welcome wizard that would probably be a good way to confirm that path is working too. Here is what the changes look like on my box:

![image](https://cloud.githubusercontent.com/assets/470472/16673945/b5fb7ed2-4467-11e6-924e-7743c250889f.png)

**Comment by Piedone** [comment]
Nice! I have a hunch that Orchard is less popular than DNN or Umbraco, but don't have any data on this. Although I'm an Orchard developer, not the maintainer of this project I'd encourage you to submit a PR.

I also know that @bleroy would be interested in this.

**Comment by bleroy** [comment]
The Live Writer team will have to decide whether they want to take this, but it looks suspect to me: the way I understand it, the list is showing APIs, not blog engines. As such, Orchard was already supported because it supports some of the APIs above, so the new entry does not necessarily add anything new. I'd like to see the pull request in order to understand what exactly it's doing.
Maybe the right change would in fact be to show a list of some of the CMS supporting each API, next to each choice. That would be more helpful to OLW users.

**Comment by rfulwell** [comment]
To be clear, this is a UI change only! Perhaps I can save you time reviewing a pull request and simply put that out there.

As a new contributor to this project, I did my best to faithfully represent the issue, as written. That is also why I included a screenshot prior to making a pull request as I don't want to waste anyone's time reviewing unneeded/unwanted changes.

This brings up some questions which I've tried to answer with a little research on OLW process (over on [Gitter](https://gitter.im/OpenLiveWriter/OpenLiveWriter), etc.):
1) Is there a label I should be looking for when issues are clearly "approved" by the Live Writer team?
2) Who are the members of the Live Writer team?
3) Do you know where I might find some examples of popular CMS that support each of the APIs available in OLW?

I would be happy to update this issue with the modified intent such that we make clear the distinction between supported APIs and actual blog engines.

Perhaps the large type question in this wizard screen should be:
"What blog service API do you use?"

I wonder if the original issue opener (do folks call that the OP around here?) would be content changing the "Other services" to "Other Metaweblog API services" and the supporting text from this:
"TypePad and more"
to this:
"Orchard, TypePad and more"

Does that sound like a more appropriate change @bleroy?

What other CMS should be listed in that "Other" category while we're at it?

**Comment by bleroy** [comment]
Yes. I don't think you can avoid going to each of the major CMS and seeing what APIs they support.

**Comment by shanselman** [comment]
Thanks for reporting this and for helping Open Live Writer over the years. We are doing an issue-bankruptcy cleanup so we can make the repository maintainable again and focus on current, actionable work. This older issue is being closed as part of that cleanup, not because it was unimportant. If this still reproduces in the current code or is still relevant to the revitalization work, please open a fresh issue with current details and we will triage it going forward.
